src/Propel/Generator/Model/ForeignKey.php

```diff
@@ -168,19 +168,20 @@ protected function doNaming()
      * Returns the normalized input of onDelete and onUpdate behaviors.
      *
      * @param string|null $behavior
+     * @param string|null $default
      *
      * @return string
      */
-    public function normalizeFKey($behavior)
+    public function normalizeFKey($behavior, $default = null)
```

Comment:
Small suggestion. You can use type-hint `?string` since it is a new code and it doesn't break BC. It is actual and for other places with new code.

Reply:
Done